### DISCUSSION TITLE
How to add URL to forgot password email? And how to custom that email?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [payloadcms/payload#1790] [Q&A] [answered]
Hi, I'm trying to send and custom email. Is there a function in PayloadCMS ready to send email
Ex: 
```
payload.email.send("abc@gmail.com",{
body: "Here is custom html",...
})
```
When I recieved email forgot password, I lost the domain, how can I config it?
![image](https://user-images.githubusercontent.com/15695252/210269914-9caebbb8-b46b-4644-bf4c-61f4035eb756.png)

**Comment by jacobsfletch** [comment, marked as answer]
Hi @IRediTOTO, yes you can send email with Payload using the `payload.sendEmail` method as long as email has been configured. Here's the docs related to that: https://payloadcms.com/docs/email/overview#email-functionality

Then you can generate your own email using `auth.forgotPassword.generateEmailHTML` on your auth collection's config — more on that here: https://payloadcms.com/docs/authentication/config#forgot-password

**Comment by mohanrajlearn** [comment]
When i enter any email id in forgot password field, that is registered or non-registered emails i am getting success message in network tab.While implementing in my nextjs project for toast container, i am always getting same success message for registered or non-registered emails.How can i determine it?
